docs/content/flags.md

```diff
@@ -616,7 +616,7 @@ and may be set in the config file.
       --webdav-pass string                           Password (obscured)
       --webdav-url string                            URL of http host to connect to
       --webdav-user string                           User name
-      --webdav-vendor string                         Name of the Webdav site/service/software you are using
+      --webdav-vendor string                         Name of the WebDAV site/service/software you are using
```

Comment:
This is an auto generated file I think

Reply:
Oh really, I didn't know that (or perhaps just completely forgot it). Thanks.